gmsh_interop/reader.py

```diff
@@ -438,32 +438,40 @@ class GmshMeshReceiverBase:
     gmsh_element_type_to_info_map: ClassVar[dict[int, GmshElementBase]] = (
         _gmsh_supported_element_type_map())
 
+    @abstractmethod
     def set_up_nodes(self, count: int) -> None:
         pass
 
+    @abstractmethod
     def add_node(self, node_nr: int, point: Point) -> None:
         pass
 
+    @abstractmethod
     def finalize_nodes(self) -> None:
         pass
 
+    @abstractmethod
     def set_up_elements(self, count: int) -> None:
         pass
 
+    @abstractmethod
     def add_element(self,
                     element_nr: int,
                     element_type: GmshElementBase,
                     vertex_nrs: IndexArray,
-                    lexicographic_nodes: Nodes,
+                    lexicographic_nodes: IndexArray,
```

Comment:
I think this was recently made into a `Nodes`? As far as I can tell (added a breakpoint in there), it gets an `int64` array of node indices:
https://github.com/inducer/gmsh_interop/blob/46f638efd7bf3c5dfcc6fbde93876c96304a5a62/gmsh_interop/reader.py#L753-L757